src/transformers/modeling_tf_utils.py

```diff
@@ -384,7 +387,7 @@ def run_call_with_unpacked_inputs(self, *args, **kwargs):
     return run_call_with_unpacked_inputs
 
 
-def input_processing(func, config, input_ids, **kwargs):
+def _input_processing(func, config, input_ids, **kwargs):
```

Comment:
Not sure the name is worth changing. The function is not imported in any init already.